app/views/local_transaction/unavailable_service.html.erb

```diff
@@ -1,17 +1,23 @@
 <%= render layout: 'shared/base_page', locals: {
-  title: "This service is not available in #{@country_name}",
+  title: @service.title,
   publication: @publication,
   edition: @edition,
 } do %>
   <div class="interaction">
-    <%= @content.html_safe %>
+    <p class="govuk-body">
+      We've matched the postcode to <span class="local-authority"><%= @local_authority.name %></span>.
```

Comment:
Should we not add this to the config too?  I guess this page will only ever be used for postcode lookup, but just thinking if anyone might want to change/alter this text.

Reply:
Yeah good question, so our currently all our unavailable pages have this on it. It's pretty generic text, which will make sense in most contexts. I've avoided putting this into config, so we don't need to support interpolation and just to keep things simple.